base/common/ya.make

```diff
@@ -1,12 +1,53 @@
 LIBRARY()
 
+ADDINCL(
+    # public
+    GLOBAL clickhouse/base
+    GLOBAL clickhouse/dbms/src
+    GLOBAL util/digest # for <city.h>
```

Comment:
Need to double-check that we are using fixed version of CityHash64 even in Arcadia.